src/utils/client.js

```diff
@@ -21,6 +21,12 @@ export const getClient = async () => {
   const root = document.querySelector('[role=application]')
   const data = root.dataset
 
+  const protocol = window.location.href.split('://')[0]
```

Comment:
You can access the protocol directly with `window.location.protocol`. It just includes a trailing `:`, so you can omit it on line 26 while initializing `cozy.client`